spec/app/ics-027-interchain-accounts/README.md

```diff
@@ -113,86 +102,119 @@ function ExecuteTx(sourcePort: string, destPort: string, destChannel: string, ms
 // * UTILITY FUNCTIONS *
 
 // Sets the active channel
-function SetActiveChannel(portId: string, channelId: string) returns (error){
+function SetActiveChannelID(portId: string, channelId: string) returns (error){
 }
 
 // Returns the id of the active channel if present
-function GetActiveChannel(portId: string) returns (string, boolean){
+function GetActiveChannelID(portId: string) returns (string, boolean){
 }
 
 // Stores the address of the interchain account in state
 function SetInterchainAccountAddress(portId string, address string) returns (string) {
 }
 
 // Gets the interchain account from state
-function GetInterchainAccountAddress(portId string) returns (string, error){
+function GetInterchainAccountAddress(portId string) returns (string, bool){
 }
 ```
 
-### Registering & Controlling flows
-**Active Channels**
+### Register & Controlling flows
+
+#### Register Account Flow
+
+To register an interchain account we require an off-chain process (relayer) to listen for `OnChannelOpenInit` events with the capability to finish a channel creation handshake on a given connection. 
+
+1. The controller chain binds a new IBC port with an id composed of the *source/counterparty connection-ids* & the *interchain account owner address*
+
+The IBC port identifier will look like this:
+```
+ics27-1.{connection-id}.{counterparty-connection-id}.{owner-address}
+```
+
+This port will be used to create channels between the controller & host chain for a specific owner/interchain account pair. Only the account with `{owner-address}` matching the bound port will be authorized to send IBC packets over channels created with `ics27-1.{connection-id}.{counterparty-connection-id}.{owner-address}`. It is up to each controller chain to enforce this port registration and access on the controller side. 
```

Comment:
`owner-address` is the account address on the controller chain, correct?

Reply:
Correct. I've been thinking lately that we should maybe use the terminology `owner-id` as it's up to the controller chain to manage this, and there's not any enforcement on the host side that this needs to be an account address. WDYT?

Reply:
Sounds fair, so long as it is consistent

Reply:
I decided to keep this as the address of the account owner on the controller chain. I've replaced any reference of `owner-address` or `owner-id` with `owner-account-address`. 

I think the opening definition for `Interchain Account Owner` should make it obvious what the `owner-account-address` is representing. WDYT?